source/_components/notify.html5.markdown

```diff
@@ -111,6 +111,22 @@ By default, every notification sent has a randomly generated UUID (v4) set as it
 }
 ```
 
+Example of adding a tag to your configuration. This won't create new notification if there already exists one with the same tag.
+
+```yaml
+  - alias: Push/update notification of sensor state with tag
+    trigger:
+      - platform: state
+        entity_id: sensor.sensor
+    action:
+      service: notify.html5
+      data_template:
+        message: "Last known sensor state is {{ states('sensor.sensor') }}."
```

Comment:
You need to embed the template with `{% raw %}{% endraw %}` otherwise it will be rendered.

Seems that this part was missing in the [website documentation](https://home-assistant.io/developers/website/).